recipe/meta.yaml

```diff
@@ -28,15 +28,15 @@ requirements:
     - ipykernel
     - ipython_genutils
     - jinja2
-    - jupyter_core >=4.6.0
+    - jupyter_core >=4.6.1
```

Comment:
@lresende anything else from the new release? (and thanks for getting it out!) I don't see much else from the [diff](
https://github.com/jupyter/notebook/compare/6.0.2...6.0.3#diff-2eeaed663bd0d25b7e608891384b7298) that is a gotcha, but you know...